solana/solana-ibc/programs/solana-ibc/src/tests.rs

```diff
@@ -575,7 +557,7 @@ fn anchor_test_deliver() -> Result<()> {
     assert_eq!(
         ((account_balance_before.ui_amount.unwrap() -
             account_balance_after.ui_amount.unwrap()) *
-            10_f64.powf(mint_info.decimals.into()))
+            10_f64.powf(9.0))
```

Comment:
```suggestion
            10_f64.powi(9))
```
would also work; or just hard code the number.

Reply:
makes sense.